controller.go

```diff
@@ -33,6 +33,9 @@ import (
 	"k8s.io/client-go/tools/record"
 	"k8s.io/client-go/util/workqueue"
 
+	// Trick dep (DO THIS BETTER)
+	_ "k8s.io/apimachinery/pkg/util/sets/types"
```

Comment:
For my edification, why is this needed?

Reply:
It isn't strictly needed **here** (and should probably move elsewhere so it isn't being linked), but we need it *somewhere* to trick `dep` into venodring this particular sub-portion of that repo (it's needed by Bazel).

Reply:
I backed this out, just to grab the location here it breaks, see [here](https://github.com/mattmoor/sample-controller/blob/bbae7b8ebb613ede9bd444e2700ee12dcedf9cf6/vendor/k8s.io/apimachinery/pkg/util/sets/BUILD#L23-L49)

Reply:
I think we can do that in dep by adding a `required` directive to `Gopkg.toml`: https://github.com/golang/dep/blob/master/docs/Gopkg.toml.md

```toml
# needed by Bazel
required = ["k8s.io/apimachinery/pkg/util/sets/types"]
```

Reply:
Submitted a PR to your fork at https://github.com/mattmoor/sample-controller/pull/1